aiotruenas_client/websockets/protocol.py

```diff
@@ -121,6 +118,10 @@ async def subscribe(self, name: str) -> asyncio.Queue:
         )
         return await sub_future
 
+    @abstractmethod
+    async def _authenticate(self):
+        """Authentication method."""
```

Comment:
Could you elaborate a little bit more here so future folks know what this method should do?